shared/chat/conversation/header-area/search.js

```diff
@@ -34,5 +37,11 @@ export default compose(
       }
     },
   }),
-  withProps({autoFocus: true, searchKey: 'chatSearch'})
-)(UserInput)
+  withProps({
+    autoFocus: true,
+    headerStyle: {borderBottomWidth: 0},
+    searchKey: 'chatSearch',
+    title: 'New Chat',
+    placeholder: 'Search someone',
```

Comment:
Does this do anything? I see `Search Keybase` in the search input.

Reply:
i saw some weird behavior on mobile where it'd appear and disappear .we'll have to dig into it later